example/lib/util/signal_level.dart

```diff
@@ -1,6 +1,25 @@
+import 'package:flutter/material.dart';
+
 enum SignalLevel { high, medium, low, unknown }
 
-SignalLevel parseRssi(int rssi) {
+extension SignalLevelExtenstion on SignalLevel {
```

Comment:
why is it placed in `utils` package? it seems to be part of the domain/model

Reply:
Moved to the `model` package